fuse/utils/data/collate.py

```diff
@@ -100,7 +100,7 @@ def _collect_values_to_list(self, samples: List[str], key: str) -> Tuple[List, b
                 if self._raise_error_key_missing:
                     raise Exception(f"Error: key {key} does not exist in sample {index}: {sample}")
                 else:
-                    value = None
```

Comment:
I thought that your use-case deals with a sample that is None and missing a key in one of the samples.
Cann you comment about it?